src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/SwitchServiceImpl.java

```diff
@@ -284,34 +285,16 @@ public CompletableFuture<SwitchValidationResultV2> validateSwitch(SwitchId switc
                                                                       String excludeString) {
         logger.info("Validate api V2 request for switch {}", switchId);
 
-        Set<IncludeFilter> includeFilters = new HashSet<>();
-        if (includeString != null) {
-            try {
-                includeFilters = switchMapper
-                        .toIncludeFilters(Arrays.stream(includeString.split("\\|"))
-                                .collect(Collectors.toList()));
-            } catch (IllegalArgumentException exception) {
-                throw new MessageException(ErrorType.REQUEST_INVALID, exception.getMessage(),
-                        "Error while parsing include parameters");
-            }
-        }
+        Set<ValidationFilter> includeFilters = parseV2ValidationFilters(includeString);
```

Comment:
Where is the "default" include filter value applied if client do not set any "include" value?

Reply:
fixed